packages/core/src/data-editor/use-column-sizer.ts

```diff
@@ -62,7 +62,9 @@ export function measureColumn(
             }
         }
     }
+    ctx.font = theme.headerFontFull;
     max = Math.max(max, ctx.measureText(c.title).width + theme.cellHorizontalPadding * 2 + (c.icon === undefined ? 0 : 28));
+    ctx.font = theme.baseFontFull;
```

Comment:
Maybe it's slightly better to save the current `ctx.font` into a variable and reset it from that:

```typescript
const currentFont = ctx.font;
ctx.font = theme.headerFontFull;
max = Math.max(max, ctx.measureText(c.title).width + theme.cellHorizontalPadding * 2 + (c.icon === undefined ? 0 : 28));
ctx.font = currentFont;
```

Reply:
Yeah, I had considered this as well. I think either could make sense in the existing code. I find it a bit odd that the caller sets `ctx.font` (for the base font), vs this function controlling that itself, but was avoiding making broader changes.

Anyway, pushed your suggestion:
https://github.com/glideapps/glide-data-grid/pull/1040/commits/62df81dfc0429d56bf9df4ca2eaf31db6927bd67